### ISSUE TITLE
[rule] Bug indenting with intercalated text node

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#58]
Code following this pattern is currently reported as invalid HTML.

```html
<div>
  foo
  <ul>
    <li>bar</li>
  </ul>
</div>
```

**Comment by KamiKillertO** [comment]
Thanks again for reporting. It's fixed and will be released soon.
As I mentioned in the previous issue, LintHTML is forked from htmllint. There's no differences between the rules except for all indent rules that I've completely rewrite 😅 .

**Comment by KamiKillertO** [comment]
@acalvo is everything ok now ?

**Comment by acalvo** [comment]
Hey! Sorry for the delay. I run the new version as soon as you released it and it seemed OK, but didn't have time to do anything else till now.

Now I went ahead with the migration and found another matter. Will report it now.

Thanks a lot for your work 😊
